docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -772,6 +772,9 @@ The output of the query has the following history columns:
 * - `version`
   - `BIGINT`
   - The version of the table corresponding to the operation
+* - `in_commit_timestamp`
+  - `TIMESTAMP(3) WITH TIME ZONE`
+  - The time when the table version became active
```

Comment:
`The time when the commit is considered to have succeeded. This is populated only when the writer has used "inCommitTimestamps" feature for the underlying table`